src/components/SetupAssistant/SetupAssistantItem.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Button } from '../Button';
+import { ProgressRing } from '../Progress';
+import { MediaObject } from '../MediaObject';
+import { SummaryDetail } from '../SummaryDetail';
+
+const SetupAssistantItem = ({
+  children,
+  className,
+  isOpen,
+  title,
+  stepProgress,
+  step,
+  renderProgress,
+  renderAddon,
+  renderOpenContent,
+  onOpen,
+  ...rest
+}) => {
+  const renderTitle = () => (
+    <div className="slds-setup-assistant__step-summary">
+      <MediaObject
+        figureLeft={renderProgress({ stepProgress, step })}
+        bodyClassName="slds-m-top_x-small"
+        center={false}
+      >
+        <MediaObject
+          figureRight={renderAddon()}
+          bodyClassName="slds-setup-assistant__step-summary-content"
+          center={false}
+        >
+          <h3 className="slds-setup-assistant__step-summary-title slds-text-heading_small">
+            {onOpen ? (
+              <Button flavor={null} className="slds-button_reset" onClick={onOpen}>
+                {title}
+              </Button>
+            ) : title}
+          </h3>
+          <p>{children}</p>
+        </MediaObject>
+      </MediaObject>
+    </div>
+  );
+
+  return (
+    <li className={cx('slds-setup-assistant__item', className)} {...rest}>
+      <article className="slds-setup-assistant__step">
+        <SummaryDetail
+          isOpen={isOpen}
+          onOpen={onOpen}
+          renderTitle={renderTitle}
+          iconButtonClassName="slds-m-top_x-small"
+        >
+          {renderOpenContent && (() => (
+            <div className="slds-setup-assistant__step-detail">
+              {renderOpenContent()}
```

Comment:
Correct me if I'm wrong, but shouldn't this receive `{ isOpen }` from parent closure now too?